.github/workflows/openqa.yml

```diff
@@ -20,9 +20,16 @@ jobs:
     container:
       image: registry.opensuse.org/devel/openqa/containers/tumbleweed:client
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Trigger and monitor openQA test
-        run: >
+        id: openqa_schedule
+        run: |
+          if [[ -z "$OPENQA_API_KEY" && -z "$OPENQA_API_SECRET" ]]; then
+            echo "Skipping openQA scheduling step: No API Keys configured."
+            echo "Configure OPENQA_API_KEY & OPENQA_API_SECRET secrets"
+            exit 0
+          fi
+
```

Comment:
If this fails without error then tests would be silently skipped. I don't think this is a good idea

Reply:
Failing it always is not a good idea either, what about scheduling only if the envs are set?

Reply:
What's the motivation to use this workflow without valid credentials? That feels wrong

Reply:
quick heads up: on Jan 14th I added an optional clause for the workflow so it's not scheduled.

https://github.com/os-autoinst/os-autoinst-distri-example/pull/29/commits/7982db8fb3016816a7181a50d0490eb72fa4f7ef#diff-de0e4ae603aa915aebe97e563c4ffe51dfe10881cc9904c4dfe0c39faebbffd6R22